### PR TITLE
docs(fury-on-vms): add sections for custom registry

### PR DESCRIPTION
**Pull request by stefanoghinelli**
Add documentation for custom registry configuration in Fury on VMs

### CODE REVIEW

**Review by ralgozino — APPROVED** [review]
The quick start guides are supposed to be well, a quick start, and not cover advanced or special use-cases like the custom registry or all the possible features.

But, being that we are lacking a more complete documentation (some progress is being done in https://github.com/sighupio/fury-distribution/pull/268). I think we can add this to the guide.

Thoughts @Rick-1990 ?

File: fury-on-vms/README.md [hunk 1]
```diff
@@ -216,6 +216,17 @@ Next we need to define the masters node and the worker nodes. The FQDN that will
 
 For example, `master1` will become `master1.example.tld`.
 
+#### Custom registry for Kubernetes core components images
+
+We can override the URL of the registry where to pull images from for the Kubernetes core components (kube-apiserver, kube-controller-manager, kube-scheduler, kube-proxy, coredns). The host is mandatory, while the port is optional. Always append `/fury/on-premises` at the end (the default value is `registry.sighup.io/fury/on-premises`).
```
Comment (ralgozino) [inline fury-on-vms/README.md:221]: Do you actually need to put the images under `/fury/on-premises` on your registry too? Can't you use a different path?

**Comment by Rick-1990** [comment]
> The quick start guides are supposed to be well, a quick start, and not cover advanced or special use-cases like the custom registry or all the possible features.
> 
> But, being that we are lacking a more complete documentation (some progress is being done in [sighupio/fury-distribution#268](https://github.com/sighupio/fury-distribution/pull/268)). I think we can add this to the guide.
> 
> Thoughts @Rick-1990 ?

I agree with both statements. We can put this in the quickstart guide for now and move it somewhere else when we refactor the docs structure, maybe in a new "Advanced Use Cases" section (where we could also put the air-gapped configuration for example)

File: fury-on-vms/README.md (hunk 1 above)
Comment (stefanoghinelli) [reply fury-on-vms/README.md:221]: You're right, the `/fury/on-premises` path is not always needed with a custom registry. It's necessary only when preserving the mirrored structure of the official registry. For others use cases a different path can be used. I propose updating the readme to reflect that.

Comment (ralgozino) [reply fury-on-vms/README.md:221]: Go ahead, feel free to push the fix to this branch. I think the same applies to the registry for the kubernetes components

**Review by ralgozino — APPROVED** [review]

**Pull request merged**